airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/copy/StreamCopier.java

```diff
@@ -36,7 +37,7 @@
   /**
    * Writes a value to a staging file for the stream.
    */
-  void write(UUID id, String jsonDataString, Timestamp emittedAt) throws Exception;
+  void write(UUID id, AirbyteRecordMessage recordMessage) throws Exception;
```

Comment:
This interface change is necessary to drop in the `S3ParquetWriter` directly for convenience.